.flake8

```diff
@@ -1,3 +1,7 @@
+# SPDX-License-Identifier: CC0-1.0
+#
+# SPDX-FileCopyrightText: Copyright 2022 Micron Technology, Inc.
```

Comment:
Is empty comment line required?  I'd just as soon put them next to each other.

Reply:
Nope. Just seems to be standard practice and is what we were doing before.